### ISSUE TITLE
Different error message for loading data from Zenodo

### ISSUE THREAD

**Issue by kc-mcp** [saeyslab/nichenetr#60]
Hi, 

I'm a first-time user of NicheNet and trying to follow the main vignette titled "NicheNet's ligand activity analysis on a gene set of interest". However, I am receiving this error message for the first command line of Step 1 in this vignette:

`ligand_target_matrix = readRDS(url("https://zenodo.org/record/3260758/files/ligand_target_matrix.rds"))`

URL 'https://zendo.org/record/3260758/files/ligand_target_matrix.rds': Timeout of 60 seconds was readError in read RDS(url("https://zenodo.org/record/3260758/files/ligand_target_matrix.rds")) : 
      cannot read from connection

I also tried downloading the data from zenodo to my local repo and received this error with the first command:

Error in gzfile(file, "rb") : cannot open the connection

Can you help me understand what is happening here and how to fix the issue? 

Thanks in advance!

**Comment by rachel662** [comment]
I have also had this problem too! would be great to know how to fix it, thank you!

**Comment by browaeysrobin** [comment]
Hi @kc-mcp and @rachel662 

It is sometimes possible that there are connection issues with Zenodo. Trying again now might work (I currently have no problems in executing the `ligand_target_matrix = readRDS(url("https://zenodo.org/record/3260758/files/ligand_target_matrix.rds"))`
 command). Another issue could be the speed/stability of your internet connection when reading in the data from Zenodo directly from within R.

About the error message after local download: are you sure that the download worked and that the way you read in the local file is correct (with correct path etc: example: ligand_target_matrix = readRDS("path/ligand_target_matrix.rds")?  )

**Comment by kc-mcp** [comment]
hi! 

@browaeysrobin I will check into the correct path for the downloaded item. However, I was able to get it to work with the url. I had to load the readr library and use read_rds function: 

`library(readr)`
`read_rds(url("https://zenodo.org/record/3260758/files/ligand_target_matrix.rds").`

So woohooo!!! But why it had to be this way is still a mystery...

Btw, @rachel662 It has also worked for all of the other zendo links as well in this vignette and the other nichenet vignettes I have done so far. Hopefully this solution will work for you too!
